README.md

```diff
@@ -1,2 +1,93 @@
-# crimpex
+# Crimpex
 Creating an md5 hash of a number, string, array, or hash in Elixir
+
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `crimpex` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:crimpex, "~> 0.1.0"}
+  ]
+end
+```
+
+Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
+and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
+be found at [https://hexdocs.pm/crimpex](https://hexdocs.pm/crimpex).
+
+# El-Crimp
```

Comment:
Crimpex?

Reply:
🤦‍♂ I did a search and replace for _ not -

Reply:
Done!